src/Components/Collapsible.lua

```diff
@@ -39,8 +39,8 @@ function Collapsible:render()
 		Topbar = e(IconListItem, {
 			Activated = self.toggle,
 			Icon = self.state.open and
-				"rbxasset://textures/collapsibleArrowDown.png" or
-				"rbxasset://textures/collapsibleArrowRight.png",
+				"rbxassetid://6028345928" or
+				"rbxassetid://6028354556",

```

Comment:
It's not obvious what either of these asset IDs are, but I definitely should be uploading them to my account. I'll set up a tarmac setup later today or tomorrow that you can merge into your own branch.